shell/platform/darwin/macos/framework/Source/FlutterMacOSCompositorUnittests.mm

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterMacOSCompositor.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/MockFlutterViewController.h"
+#import "flutter/testing/testing.h"
+
+namespace flutter::testing {
+
+TEST(FlutterMacOSCompositorTest, TestPresent) {
+  id mockViewController = CreateMockViewController(nil);
+
+  flutter::FlutterMacOSCompositor* macos_compositor = new flutter::FlutterMacOSCompositor(nullptr, mockViewController);
+
+  bool* flag = new bool(false);
```

Comment:
make this `unique_ptr<bool>` and call `reset`

Reply:
Changed to a regular bool instead.